src/content/structured/get-started/testing-with-jest-and-rtl.mdx

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
The links to the 'Testing with Jest' and 'Testing with React Testing Library' pages on the testing Overview page will need updating (one link pointing to the 'Testing with Jest and React Testing Library' page).